include/StepRecorder.h

```diff
@@ -66,11 +66,6 @@ class StepRecorder : public QObject
 		return m_isRecording;
 	}
 
-	QColor curStepNoteColor() const
-	{
-		return QColor(245,3,139); // radiant pink
-	}
-
```

Comment:
I moved this to under PianoRoll. Is this acceptable?

Reply:
`PianoRoll.cpp` is already too messy but if this function should be there, I can't say anything else. 

Maybe @Veratil can say anything more on this

Reply:
Yeah, I'm looking at moving it back right now.

Reply:
Is it used anywhere else other than PianoRoll? If so, keep it here. If not, it's fine to move.

Reply:
I'm not sure I follow you. It belongs to the step editor but is only called in the Piano Roll. I moved it because it simplified the coding when testing things out and it felt natural to keep it there.

Reply:
If it's only called in the PianoRoll then it should belong to the PianoRoll. Moving it there is fine.

Reply:
Ah, OK, thankx! Fixing!

Reply:
Fixed!